public/index.html

```diff
@@ -40,5 +38,22 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
```

Comment:
Am not sure it's going to be a good idea to bundle jquery and React in the same application, but even then, It would be better to take a module bundling strategy since we already have webpack in place to reduce the number of HTTP cycles the browser has to make to get all these resources. With Http2 this is not going to be an issue but we are not there yet..